guides/v2.1/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -108,14 +110,19 @@ Use this table to understand what changes Magento can make and which version num
 | | Changed event property | MAJOR|
 | | Removed event property | MAJOR|
 | | Changed event ordering | MAJOR|
-| **Virtual Type**| Virtual type removed | MAJOR|
+| **Virtual Type (marked with `@api`)**| Virtual type removed | MAJOR|
 | | Virtual type added | MINOR|
-| **URL Paths** | Path removed | MAJOR|
-| | Removed/renamed a request parameter| MAJOR|
-| | New required request parameter | MAJOR|
-| | New optional request parameter | MINOR|
+| **URL Paths (Store-Front and Admin)** | Path removed | MAJOR|
```

Comment:
Do we use PUT/UPDATE/DELETE endpoints anywhere?

Reply:
I think in WebAPI only. It's not covered by these items.